test/unit/util.test.ts

```diff
@@ -65,3 +67,45 @@ test('Get full URL', async(t) => {
             'https://wikimedia.org/api/rest_v1/media/math/render/svg/34cbb1e27dae0c04fc794a91f2aa001aca7054c1',
             'Full Url when base and url both strtas with http/s');
 })
+
+test('Custom flavour path', async(t) => {
+
+    // checks in current working directory.
+    let createStream = fs.createWriteStream(pathParser.resolve(process.cwd(), 'testCustomFlavour.js'));
```

Comment:
In general, you should avoid to write randomly in the current directory or in the code directory. You never know what kind of side effect you could have (like the file you want to create exists already, people works during the test and will see file appearing/disappearing). I fyou have to create tmp files. You create them in the OS temporary directory and you secure that even if the test for some reason is stopped in the middle then the files disappear by using something like https://www.npmjs.com/package/tmp. The good news is that I don't believe you need that at all here.

Reply:
@kelson42 this test is for case one(searching for file in current wokring directory) so I guess this file is needed and we would need tmp folder as well then.

Reply:
OK, then `tmp` package to create a directory in OS temporary directory and inside put your fake extension (with `tmp`)... and do whatever is necessary.